test/check-multi-os

```diff
@@ -99,7 +104,7 @@ class TestMultiOS(MachineCase):
             stock_m.wait_boot()
             stock_b = self.new_browser(stock_m.address)
 
-            stock_b.login_and_go("dashboard", href="/dashboard/list", host=None)
+            stock_b.login_and_go("/dashboard/list", host=None)
```

Comment:
The stock version of the dashboard still needs the old way, since it hasn't changed.